Gemfile

```diff
@@ -42,7 +42,9 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+# TODO(amos): Unfork this gem: https://github.com/18F/identity-private/issues/708
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
Sorry to throw more delays in there, but I would love to see this land without a change to this gem.  I spoke to @monfresh and we think it should be possible.

For avoiding OTP on login, I think we can just override `user#send_new_otp` to do nothing and use a new method with the same two lines to do the sending.   Its not pretty, but i think @monfresh as other plans for the upstream gem to make the auto-sending more configurable by moving it into the controller.